DOCS.md

```diff
@@ -0,0 +1,29 @@
+Use this plugin to publish artifacts from the build to an SFTP server
+You can override the default configuration with the following parameters:
+
+* `host` - Server host
+* `port` - Server port, default to 22
+* `username` - Server username, default to blank
+* `password` - Password for password-based authentication
+* `privateKey` - String containing the private key for key-based authentication
```

Comment:
also drone will automatically provide a public / private key pair for each repository (viewable in the repository settings screen in the drone UI). This is provided in `workspace.keys.private` ... for may of the other plugins (like rsync and ssh) we use this key by default.

for reference https://github.com/drone-plugins/drone-rsync/blob/master/main.go#L202